templates/partials/latest.php

```diff
@@ -20,7 +20,7 @@
 		<a class="timemanager-pjax-link" href="<?php echo $urlGenerator->linkToRoute('timemanager.page.times'); ?>?task=<?php echo $entry->task->getUuid(); ?>">
 			<h3><?php p($entry->client->getName()); ?> › <?php p($entry->project->getName()); ?> › <?php p($entry->task->getName()); ?></h3>
 			<div class="tm_item-excerpt">
-				<span><?php p($entry->time->getStartLocalized()); ?></span>
+				<span data-datetime="<?php p($entry->time->getStartFormatted("Y-m-d H:i:s")); ?>"><?php p($entry->time->getStartLocalized()); ?></span>
```

Comment:
Interesting! Will this help people with screenreaders to get the dates in their preferred format? Maybe there's a different function I don't know about? 😊

Reply:
Possibly? The actual intention (I added some comment for this in [helpers.js:136](https://github.com/devattendant/timemanager/blob/master/js/lib/helpers.js#L136)) was to easily find all displayed all UTC dates and transform them into local dates in JS code.